data_steward/cdr_cleaner/cleaning_rules/drop_duplicate_states.py

```diff
@@ -0,0 +1,88 @@
+"""
+ Currently there are some individuals with multiple state records from consent.
+ We need to include only one state, dropping all but the most recent record.
+"""
+import os
+
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+import sandbox
```

Comment:
This import is not used by the module unless the module is run as a stand alone script.  You should move this import to line 81, following `import cdr_cleaner.clean_cdr_engine as clean_engine`.  Then it will only be imported if this is run as a stand alone script.